### DISCUSSION TITLE
「开源之史」系列之一/九：开源的治理之道——非营利组织与解决难题 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#41] [Announcements]
# 「开源之史」系列之一/九：开源的治理之道——非营利组织与解决难题 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/07-os-organzition-and-governance/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

NGO 和 FLOSS 本质逻辑不同哈,
公地悲剧 最本质的原因是, 成本过小, 嘦将自己的羊拉到公地即可;

但是, FLOSS 软件的使用是有成本/门槛的, 如果没有一定知识, 根本无从下手,
除非是面向终端用户的开源作品,
否则, 多数 FLOSS 工程都是为了解决开发者自身具体问题而构建的, 
多数普通人/企业/...根本没有 迁来就能吃到草的羊...

能消化 FLOSS 营养的羊, 一定来源开源社区,
这其实就开源文化生存之道了:

- 通过开源社区接触开源文化
- 通过开源文化快速获得成长
- 通过成长获得商业成功...这时, 才能有效回馈开源

通过什么呢?

- 直接商业
- 直接捐助
- 间接捐助
- 间接商业
- ...

甚至于, 单单公开承认用了什么开源项目就已经是有效回馈了;

所以, FLOSS 的非营利和 NGO 的完全不同, 难以类比...

最关键的, NGO 中每个义士都是纯粹的物资消解单位,
而 FLOSS 中每个合格的开发者, 都是持续生产单位.

**Reply by lijiangsheng1** [reply]
Python 软件基金会的作用是什么？ 我觉得背后一定有更深次的缘由的，不可能无缘无故成立这么个组织。

要知道，这是个和开源共同体（community）完全不同的人类组织机构。
